utils/configure-project

```diff
@@ -1,21 +1,22 @@
-#!/usr/bin/env sh
+#!/usr/bin/env bash
 
 set -eu
 
 fatal() {
-    printf "$*\n" > /dev/stderr
+    echo "$*" >&2
```

Comment:
Also no, `printf` is there for a reason. And `/dev/stderr` is way more explicit.